.fixtures.yml

```diff
@@ -2,10 +2,7 @@ fixtures:
   repositories:
     stdlib:  "git://github.com/puppetlabs/puppetlabs-stdlib.git"
     foreman: "git://github.com/theforeman/puppet-foreman.git"
-    concat:
-      repo:        "git://github.com/ripienaar/puppet-concat.git"
-      ref:         "04356974f72b90a1d0f57346a00e95a717924e43"
-    concat_native: "git://github.com/theforeman/puppet-concat.git"
+    concat:           'git://github.com/puppetlabs/puppetlabs-concat'
```

Comment:
Crazy nitpicky, but standardize on one kind of quotes, and either one space or aligned indentation the values